docker-compose.yml

```diff
@@ -72,6 +72,7 @@ services:
 
   job-consumer:
     build: ./docker
+    user: 1000:1000
```

Comment:
The `docker` build here is on buster?
https://github.com/iFixit/pim-community-standard/blob/8fc949b31e6efcbd005585a4192d674d3eaf6c26/docker/Dockerfile#L1